### DISCUSSION TITLE
Is it possible to trigger action on user's reply?

### DISCUSSION THREAD

**Discussion by jessicalu2016** [ical4j/ical4j#775] [Q&A]
Hi,

Just curious if there's a way to associate an action (e.g. open a url) when user click "Yes" or "No" as reply to the calendar invitation?

Thanks,
Jessica

**Comment by benfortuna** [comment]
I guess it depends how you are presenting the invitation to the user. If they receive it in Outlook (for example) then no you can't open a URL in their browser. 

If you control the interface by which they accept the invitation (e.g. a custom web application) then potentially you could open a URL in a new tab.

I think probably the most common approach would be to wait for invitation acceptance (e.g. via email) and then send another email with an acceptance URL.

**Comment by jessicalu2016** [comment]
Hi Ben Fortuna,

Thanks for fast reply! No, we don't control the interface. just whatever email client (Outlook, gmail etc.). I was just wondering if icalendar natively support this. seem not.

> wait for invitation acceptance (e.g. via email) and then send another email with an acceptance URL.

yeah, that's my next step if icalendar doesn't support.

Thank you!
